controllers/tasksController.js

```diff
@@ -0,0 +1,47 @@
+const taskQuery = require('../models/tasks')
+/**
+ * Creates new task
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - Task object
+ * @param res {Object} - Express response object
+ */
+const addNewTask = async (req, res) => {
+  try {
+    const task = await taskQuery.addTask(req.body)
+    return res.json({
+      message: 'Task created successfully!',
+      task: req.body,
```

Comment:
We shouldn't return the request body here. All fields from the request body are not to be added to the DB.
We should return the data stored in DB by the model function.
Can you check what data is obtained at https://github.com/Real-Dev-Squad/website-backend/pull/124/files#diff-aa74c20d1c66caed73c8f28f809dff80428137d5f605696c02ac8494c02bd46fR12 and utilise it?

Reply:
Thanks, I have updated the logic accordingly.